src/v2/Apps/Conversation/Components/ConfirmArtworkButton.tsx

```diff
@@ -2,65 +2,62 @@ import { useState } from "react"
 import * as React from "react"
 import { createFragmentContainer, graphql, RelayProp } from "react-relay"
 import { useTracking } from "react-tracking"
-import { Button } from "@artsy/palette"
+import { Button, ButtonProps } from "@artsy/palette"
 import { TappedConfirmArtwork } from "@artsy/cohesion"
 
 import createLogger from "v2/Utils/logger"
 import { MakeInquiryOffer } from "../Mutation/MakeInquiryOfferMutation"
+import { MakeInquiryOrder } from "../Mutation/MakeInquiryOrderMutation"
 
 import { ConfirmArtworkButton_artwork } from "v2/__generated__/ConfirmArtworkButton_artwork.graphql"
 
 const logger = createLogger("Conversation/Components/ConfirmArtworkButton.tsx")
 
-export interface ConfirmArtworkButtonProps {
+export interface ConfirmArtworkButtonProps extends ButtonProps {
   artwork: ConfirmArtworkButton_artwork
   relay: RelayProp
   editionSetID: string | null
-  disabled?: boolean
+  createsOfferOrder?: boolean
   conversationID: string
-  children?: React.ReactNode
   trackingEvent?: TappedConfirmArtwork
 }
 
-export const ConfirmArtworkButton: React.FC<ConfirmArtworkButtonProps> = props => {
-  const [
-    isCommittingCreateOfferOrderMutation,
-    setIsCommittingCreateOfferOrderMutation,
-  ] = useState(false)
+export const ConfirmArtworkButton: React.FC<ConfirmArtworkButtonProps> = ({
+  relay,
+  artwork,
+  editionSetID,
+  conversationID,
+  trackingEvent,
+  createsOfferOrder = true,
+  disabled,
+  variant,
+  children,
+}) => {
+  const tracking = useTracking()
+  const [isCommittingMutation, setIsCommittingMutation] = useState(false)
 
+  const commitMutation = createsOfferOrder ? MakeInquiryOffer : MakeInquiryOrder
```

Comment:
The important change on this file is that now it's able to create both buy and offer orders.